### PR TITLE
Home: Masthead v1

### LINKED ISSUE #1: Home - Masthead
Develop the home masthead for the blog according to our design sketch.

**Requirements**
- Make sure the view looks good on mobile, tablet, and mobile screens.
- Make sure the images used are optimized by using the nextjs Image component. 
- Add a heading with the copy "Fresh Beginnings".
- Add a subheading with the copy "Journey to living a toxin-free life".
- On mobile screens add a button that says about me that will eventually self-scroll the user to the about me section.

**Design Sketch**

Image

Image

### PR DESCRIPTION
**Pull request by Diabeticodes**
# Why
In generality a new branch needed made from development and synced to include initial changes from the 'mike-dev' branch. This also included the list of changes requested in the original PR...all of which have been added in the following section below.

## What
- Import statements reorganized
- Logo changed to rectangle with text and bg color instead of image
- Margin adjusted between ImageMasthead and LatestPosts component
- Removed About and Blog page since they've been re-structured in the project.

## Testing Notes (if applicable)
There is a certain smaller resolution in which the logo isn't completely centered. May need to double check to make sure it's not a common device screen size and more of a 'gray area' in which the site wouldn't normally be viewed.

## Checklist

- [X] Looks good on large screens
- [X] Looks good on mobile screens

### CODE REVIEW

**Review by giantrotta24 — COMMENTED** [review]
Not a big deal because it's just the two of us but just some pointers around PRs. Again, your description overall following the template is fantastic but you did not name your PR, so it took the branch name as default which wouldnt be super helpful to someone just scanning through PRs quickly.

**Review by giantrotta24 — COMMENTED** [review]
Another pointer, you are merging to master which does not follow our merging structure. Your development branches should be making their way to the `development` branch.

And finally, if you felt it was better to start fresh with a new branch/PR then it is always good practice to close the previous PR with a relevant comment.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
 import type { NextPage } from 'next';
 
-import HomeMasthead from '../components/Home/HomeMasthead';
-import LatestPosts from '../components/Home/LatestPosts';
-import AboutMe from '../components/Home/AboutMe';
-import ContactMe from '../components/Home/ContactMe';
+import HomeMasthead from '../components/HomeMasthead';
+import LatestPosts from '../components/Home/LatestPosts/index';
+import Contact from '../components/Home/ContactMe'
```
Comment (giantrotta24) [inline src/pages/index.tsx]: The about me section you are removing here is the one we want to keep, that is the one I merged in.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,2013 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (giantrotta24) [inline yarn.lock]: looks like you ran yarn instead of npm, you'll want to make sure this file is not committed to git history

File: src/components/Home/ContactMe/ContactMeForm.tsx [hunk 3]
```diff
@@ -5,7 +5,17 @@ const ContactMeForm: React.FC = () => {
   const { register, handleSubmit } = useForm();
 
   const onSubmit = (data: any) => {
-    console.log(data);
+    //Attempt to store input data with type assertion
```
Comment (giantrotta24) [inline src/components/Home/ContactMe/ContactMeForm.tsx]: Can we revert this change and move this to a new branch?

File: src/components/common/Logo.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import Image from 'next/image';
+
+import LogoImage from '/public/images/fresh-beginnings-logo.png';
+
+const Logo = () => {
+  return (
+    <div className="relative bottom-36 lg:left-96 md:left-0">
+      <Image
```
Comment (giantrotta24) [inline src/components/common/Logo.tsx:8]: Logo looks great! Can we adjust a few things? There is a typo, should be "Toxin free" not "Toxic", can we adjust the border radius so it's not curved (nothing in our design is curved so we want to keep that standard), and not sure if this is using our default font, which should be `font-family: ui-serif, Georgia, Cambria, "Times New Roman", Times, serif;`

**Review by giantrotta24 — CHANGES_REQUESTED** [review]
Great work, almost there! just a few things we need to change to keep our git history clean and some minor tweaks to the logo. 

Also, if you don't want to keep messing with the logo in photoshop im pretty sure you could make the same thing happen with code since it's just a square with text ;) but that's definitely not necessary, your call!

File: src/pages/index.tsx (hunk 1 above)
Comment (Diabeticodes) [reply src/pages/index.tsx]: Added

File: src/components/Home/ContactMe/ContactMeForm.tsx (hunk 3 above)
Comment (Diabeticodes) [reply src/components/Home/ContactMe/ContactMeForm.tsx]: reverted

**Review by giantrotta24 — APPROVED** [review]
Great work! Lets merge this during our next pairing session!
